js/ui/widget/ui.search_box_mixin.js

```diff
@@ -68,6 +69,21 @@ module.exports = {
         });
     },
 
+    _initDataSource: function() {
+        this.callBase();
+
+        if(this._applySourceOptions()) {
+            this.option("searchValue").length && this._dataSource.searchValue(this.option("searchValue"));
```

Comment:
set all dataSource options without condition

`this._dataSource.searchValue(this.option("searchValue")`